docs/integrations/databases/open-telemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+id: redis
+title: Sumo Logic Open Telemetry App for Redis
+sidebar_label: Redis
+description: Documentation for Open Telemetry App for Redis.
```

Comment:
Can you please write the description that should say about the content of this doc?

Reply:
OpenTelemetry should be one word.